apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -0,0 +1,140 @@
+	// a more granular application of the policy.
+	//
+	// Support: Extended for Kubernetes Service
+	//
+	// Support: Implementation-specific for any other resource
+	//
+	TargetRef PolicyTargetReferenceWithSectionName `json:"targetRef"`
+
+	// TLS contains backend TLS policy configuration.
+	TLS BackendTLSPolicyConfig `json:"tls"`
+}
+
+// BackendTLSPolicyConfig contains backend TLS policy configuration.
+// +kubebuilder:validation:XValidation:message="must not contain both CACertificateRefs and WellKnownCACerts",rule="!(has(self.caCertificateRefs) && size(self.caCertificateRefs) > 0 && has(self.wellKnownCACerts) && self.wellKnownCACerts != \"\")"
+// +kubebuilder:validation:XValidation:message="must specify either CACertificateRefs or WellKnownCACerts",rule="(has(self.caCertificateRefs) && size(self.caCertificateRefs) > 0 || has(self.wellKnownCACerts) && self.wellKnownCACerts != \"\")"
+type BackendTLSPolicyConfig struct {
+	// CACertificateRefs contains one or more references to Kubernetes objects that
+	// contain a PEM-encoded TLS CA certificate bundle, which is used to
+	// validate a TLS handshake between the Gateway and backend Pod.
+	//
+	// If CACertificateRefs is empty or unspecified, then WellKnownCACerts must be
+	// specified. Only one of CACertificateRefs or WellKnownCACerts may be specified,
+	// not both. If CACertifcateRefs is empty or unspecified, the configuration for
+	// WellKnownCACerts MUST be honored instead if supported by the implementation.
+	//
+	// References to a resource in a different namespace are invalid for the
+	// moment, although we will revisit this in the future.
+	//
+	// A single CACertificateRef to a Kubernetes ConfigMap kind has "Core" support.
+	// Implementations MAY choose to support attaching multiple certificates to
+	// a backend, but this behavior is implementation-specific.
+	//
+	// Support: Core - An optional single reference to a Kubernetes ConfigMap,
+	// with the CA certificate in a key named `ca.crt`.
+	//
+	// Support: Implementation-specific (More than one reference, or other kinds
+	// of resources).
+	//
+	// +kubebuilder:validation:MaxItems=8
+	// +optional
+	CACertificateRefs []v1beta1.LocalObjectReference `json:"caCertificateRefs,omitempty"`
+
+	// WellKnownCACerts specifies whether system CA certificates may be used in
+	// the TLS handshake between the gateway and backend pod.
+	//
+	// If WellKnownCACerts is unspecified or empty (""), then CACertificateRefs
+	// must be specified with at least one entry for a valid configuration. Only
+	// one of CACertificateRefs or WellKnownCACerts may be specified, not both.
+	// If an implementation does not support the WellKnownCACerts field or the value
+	// supplied is not supported, the Status Conditions on the Policy MUST be
+	// updated to include an Accepted: False Condition with Reason: Invalid.
+	//
+	// Support: Implementation-specific
+	//
+	// +optional
+	WellKnownCACerts *WellKnownCACertType `json:"wellKnownCACerts,omitempty"`
```

Comment:
```suggestion
	WellKnownCACertificates *WellKnownCACertType `json:"wellKnownCACertificates,omitempty"`
```

(Also would want corresponding updates to comments above)